packages/esm-patient-search-app/src/patient-search-page/patient-banner/contact-details/contact-details.test.tsx

```diff
@@ -0,0 +1,87 @@
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+import ContactDetails from './contact-details.component';
+import { useRelationships } from './relationships.resource';
+import { usePatientContactAttributes } from '../hooks/usePatientAttributes';
+
+const mockUseRelationships = useRelationships as jest.Mock;
+const mockUsePatientContactAttributes = usePatientContactAttributes as jest.Mock;
+
+const mockContactAttributes = [{ attributeType: { display: 'Email' }, value: 'test@example.com', uuid: 'email-uuid' }];
+
+const mockRelationships = [
+  {
+    uuid: 'relation-uuid',
+    display: 'John Doe',
+    relationshipType: 'Family',
+    relativeAge: 30,
+  },
+];
+
+jest.mock('./relationships.resource');
+jest.mock('../hooks/usePatientAttributes');
+
+mockUsePatientContactAttributes.mockReturnValue({
+  isLoading: false,
+  contactAttributes: mockContactAttributes,
+});
+
+mockUseRelationships.mockReturnValue({ isLoading: false, data: mockRelationships });
+
+describe('ContactDetails', () => {
+  beforeEach(() => {
+    mockUsePatientContactAttributes.mockReturnValue({
+      isLoading: false,
+      contactAttributes: mockContactAttributes,
+    });
+
+    mockUseRelationships.mockReturnValue({ isLoading: false, data: mockRelationships });
+  });
+  it('renders address', () => {
+    const mockAddress = {
+      postalCode: '12345',
+      address1: '123 Main St',
+      cityVillage: 'City',
+      stateProvince: 'State',
+      country: 'Country',
+      preferred: true,
+    };
```

Comment:
Shall we move this to mocks so we can reuse it?

Reply:
done @anjula-sack